### PR TITLE
Update README to make it more readable (Fixes #460)

### LINKED ISSUE #460: Restructure README.md
**Is your feature request related to a problem? Please describe.**
Restructure README.md for visitors to easily follow. 

**Describe the solution you'd like**
Some content can be moved to more relevant locations and links to them can be added to the README.md

**Describe alternatives you've considered**
- Move Contributors section in the README.md to CONTRIBUTING.md, perhaps.
- Put 'Additional setup instructions for Ubuntu' under the SETUP section as subheading or extension rather than a separate topic itself.
- Other restructure requests may follow as comments on this issue. To discuss further, join our slack.

**Additional context**
Intended to be solved under Google Code-in 2019.

**Are you working on this? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by JoshHeng**
The README has been restructured and edited to make it simpler to understand. Information has also been transferred to the CONTRIBUTING and SETUP files to retain simplicity.

Fixes #460 

Refer to `README.md` to see the changes (or alternatively look at this branch in my personal fork to see it nicely formatted)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,18 +1,15 @@
-## Contributing to CircuitVerse
+# Contributing to CircuitVerse
+[Back to `README.md`](README.md)
 
-- Thanks for checking out our guide. We'll be glad to hear from you.
-- Interested in making a contribution? Read on!
+Thanks for checking out our contribution guide, and we appreciate any help that you give towards CircuitVerse, no matter how small.
```
Comment (sakshi1499) [inline CONTRIBUTING.md]: Can you suggest some other way of saying it. Any help that you give sounds strange

File: code-of-conduct.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+[Back to `README.md`](README.md)
 
 As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
 
 We are committed to making participation in this project a harassment-free experience for everyone, regardless of the level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.
 
 Diversity and inclusion make the CircuitVerse community strong. We encourage participation from the most varied and diverse backgrounds possible and want to be very clear about where we stand.
 
-## Expected Behaviour
+We encourage people to talk on public platforms for maximum participation. Whenever possible, avoid taking conversations to private channels. Keeping communication public means everybody can benefit and learn from the conversation.
```
Comment (sakshi1499) [inline code-of-conduct.md]: The last point seems like its a part of expected behavior only.

**Comment by sakshi1499** [comment]
Looks good @JoshHeng💯

**Comment by MrBartusek** [comment]
I think that badges setup will be better
image image image image image
appveyor seams broken for some reason but I don't know how to fix it
``` markdown
![image](https://opencollective.com/CircuitVerse/all/badge.svg?label=financial+contributors)
![image](https://img.shields.io/circleci/build/github/CircuitVerse/CircuitVerse/master?label=circleci)
![image](https://img.shields.io/appveyor/ci/CircuitVerse/CircuitVerse?label=appveyor)
![image](https://img.shields.io/badge/chat-on_slack-purple.svg)
![image](https://img.shields.io/badge/mailing_list-join-orange)
```

**Comment by JoshHeng** [comment]
> 
> 
> I think that badges setup will be better
> image image image image image
> appveyor seams broken for some reason but I don't know how to fix it
> 
> ```gfm
> image
> image
> image
> image
> image
> ```

Most of the badges have been implemented. AppVeyor does not seem to be detecting CV's repo so I'm just not adding it

**Comment by Ananthavijay** [comment]
The appveyor fails because of bootsnap.

**Comment by Ananthavijay** [comment]
For CircleCI maybe use this
[CircleCI](https://circleci.com/gh/CircuitVerse/CircuitVerse)
````
[![CircleCI](https://circleci.com/gh/CircuitVerse/CircuitVerse.svg?style=svg)](https://circleci.com/gh/CircuitVerse/CircuitVerse)
````

**Comment by Ananthavijay** [comment]
For slack use this:

[chat : on slack](https://join.slack.com/t/circuitverse-team/shared_invite/enQtNjc4MzcyNDE5OTA3LTdjYTM5NjFiZWZlZGI2MmU1MmYzYzczNmZlZDg5MjYxYmQ4ODRjMjQxM2UyMWI5ODUzODQzMDU2ZDEzNjI4NmE)

`````
[![chat : on slack](https://img.shields.io/badge/chat-on_slack-purple.svg)](https://join.slack.com/t/circuitverse-team/shared_invite/enQtNjc4MzcyNDE5OTA3LTdjYTM5NjFiZWZlZGI2MmU1MmYzYzczNmZlZDg5MjYxYmQ4ODRjMjQxM2UyMWI5ODUzODQzMDU2ZDEzNjI4NmE)
`````
- When the user clicks on the badge, it takes the user to the slack room.

**Comment by JoshHeng** [comment]
> For CircleCI maybe use this
> [CircleCI](https://circleci.com/gh/CircuitVerse/CircuitVerse)
> 
> ```
> [CircleCI](https://circleci.com/gh/CircuitVerse/CircuitVerse)
> ```

I don’t believe we should use this shield as it doesn’t fit in with the format of the others

**Comment by thiefie** [comment]
image
image
image
image
image

Cmon Tell me who does not like square icons???

**Comment by Ananthavijay** [comment]
Yea, I also wanted to say to make the badges more modern. @thiefie Looks great.

**Comment by JoshHeng** [comment]
> image
> image
> image
> image
> image
> image
> 
> Cmon Tell me who does not like square icons???
> @JoshHeng @MrBartusek @Ananthavijay



> image
> image
> image
> image
> image
> image
> 
> Cmon Tell me who does not like square icons???
> @JoshHeng @MrBartusek @Ananthavijay

Is that still Shields.io? Can you give some example code? Looks good!

**Comment by thiefie** [comment]
@JoshHeng Yup
`image
image
image
image
image
image`

**Comment by JoshHeng** [comment]
> 
> 
> @JoshHeng Yup
> `image image image image image image`

Thanks, changes implemented

**Comment by Ananthavijay** [comment]
@JoshHeng Please add something like this ````First Create Google, Facebook, Github app by following the steps mentioned [Here](https://github.com/CircuitVerse/CircuitVerse/wiki/Create-Apps).````  in ```Setup.md```

**Comment by JoshHeng** [comment]
> 
> 
> @JoshHeng Please add something like this `First Create Google, Facebook, Github app by following the steps mentioned [Here](https://github.com/CircuitVerse/CircuitVerse/wiki/Create-Apps).` in `Setup.md`

Done

**Comment by Ananthavijay** [comment]
Thanks @JoshHeng

**Pull request merged**